src/components/UploadField.jsx

```diff
@@ -0,0 +1,72 @@
+import React, { Component } from 'react';
+import { withFirebase } from 'react-redux-firebase';
+import Dropzone from 'react-dropzone'; // For Dropzone reference check https://github.com/react-dropzone/react-dropzone
+
+@withFirebase
+class UploadField extends Component {
+  state = {
+    isLoading: false,
+    files: [],
+  }
+
+  constructor(props) {
+    super(props);
+
+    // Path in the Storage where the files will get saves
+    this.storagePath = this.props.storagePath;
+
+    // Path in the db where the files will get saved
+    this.path = this.props.path;
+
+    // Bind
+    this.onDrop = this.onDrop.bind(this);
+  }
+
+  onDrop(files) {
+
+    // Set Loading
+    this.setState({ isLoading: true });
+
+    // Upload files
+    this.props.firebase
+      .uploadFiles(this.storagePath, files, this.path)
+      .then(files => {
+
+        // Create an array with the data we need from the uploaded files
+        const uploadedFiles = files.map( file => {
+          const { key, File: { downloadURL, fullPath, name } } = file;
```

Comment:
are there other values returned here that we should save like mini type or whatever?

Reply:
That's pretty much it:
```
{
  File : {
    createdAt: {.sv: "timestamp"},
    downloadURL: "https://firebasestorage.googleapis.com/v0/b/igv-andamiaje-co.appspot.com/o/uploads%2FScreen%20Shot%202018-01-25%20at%207.18.56%20PM.png?alt=media&token=c8e32140-fb92-419f-8e8b-c7df69a07ecf",
    fullPath: "uploads/Screen Shot 2018-01-25 at 7.18.56 PM.png",
    name: "Screen Shot 2018-01-25 at 7.18.56 PM.png",
  }
  key: "-L3niEFJMApjo06GeIlw"
  snapshot: {},
  uploadTaskSnaphot: {}
  metaDataSnapshot: {}
}
```